storage/storage-samples/storage-owner-sample/src/main/java/org/fido/iot/storage/OwnerDbManager.java

```diff
@@ -65,6 +64,16 @@ public void createTables(DataSource dataSource) {
 
       stmt.executeUpdate(sql);
 
+      sql = "CREATE TABLE IF NOT EXISTS "
+              + "TO2_SETTINGS("
+              + "ID INT NOT NULL, "
+              + "WGET_SVI_MOD_VERIFICATION CHAR(5), "
```

Comment:
Would suggest to make this a boolean column instead of char.

Reply:
Done